packages/suite-desktop/scripts/build.js

```diff
@@ -7,10 +7,25 @@ const pkg = require('../package.json');
 
 const electronSource = path.join(__dirname, '..', 'src-electron');
 const isDev = process.env.NODE_ENV !== 'production';
+const useMock = process.env.USE_MOCK !== undefined;
```

Comment:
In the docs the env var is said to be in plular `USE_MOCKS` but here it's singular. That should be aligned. I'd prefer the plural form.

Optimally the plural would be used for the js variable as well, arriving at
```suggestion
const useMocks = process.env.USE_MOCKS !== undefined;
```

But see also my other remark below before applying that suggesstion. .)

Reply:
Addressed in 4200d30.

Reply:
🦊